### ISSUE TITLE
Server and Client installation question on full mesh 

### ISSUE THREAD

**Issue by bmullan** [SFTtech/wirespider#127]
I wanted to ask if the Server and Client software could be installed on multiple compute "nodes"

_In my use-case test example:_
I use Ubuntu 22.04 machines on multiple Clouds (Digital Ocean here in US and in a Hetzner DC in Germany)
I use LXD "system" Containers each also running Ubuntu 22.04 but could be many diff distro's.

By default, the LXD Containers on any one "Host" server are behind a 10.x.x.x. dnsmasq Bridge (default name "lxdbr0")

My use-case is to interconnect all the local/remote "Host" LXD containers via a mesh vpn environment.

My question is - 

- Can **_Both_** the Server & Client be installed **_on each Cloud Host server_**?
- Should they be?
- Is that how wirespider "_Server_" Redundancy might be provided?

Basically, can there be more than 1 "Server" deployed?

**Comment by bmullan** [comment]
? any response ?
